### ISSUE TITLE
Qt client: the destination is not accessible

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#1661]
### Description

so while downloading a file, in that window, you can press tab and review some information such as file name, file size, and throo put: but there is also a destination that shows where the file is downloading, but we can't look at this destination using tab / shift tab keys, i saw this one only with nvda object navigation: so please make this one accessible for tab and shift tab keys so we can review it if we want too: windows 10: Qt teamtalk vertion: 5.12

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by amirmahdifard** [comment]
@CoBC really thank you!
